vrecord

```diff
@@ -210,6 +210,47 @@ _get_decklink_inputs(){
     fi
 }
 
+_gtk_vbox_list() {
+    VARIABLE_NAME="${1}"
+    local -n OPTION_LIST="${2}"
```

Comment:
Not merging yet, since this use of `local -n` or `declare -n` requires bash 4.3+ which isn't a stated dependency